mne/viz/evoked.py

```diff
@@ -1881,6 +1882,23 @@ def _title_helper_pce(title, picked_types, picks, ch_names, combine):
     return title
 
 
+def _get_evokeds_from_epochs(instances, Epochs):
+    """Transform a dict of lists of Epochs into a dict of lists of Evokeds."""
+    if all(isinstance(content[0], Epochs) for content in instances.values()):
+        print(instances)
+        instances = {key: list(epochs[key].iter_evoked())
+                     for key, epochs in instances.items()}
+    elif all([all(isinstance(content, Epochs) for sublist in instances.values()
+                  for content in sublist)]):
+        instances = {key: [epochs[key].average() for epochs in epochs_list]
+                     for key, epochs_list in instances.items()}
```

Comment:
I think for this `if ... elif` you might consider a different strategy: check whether each entry is `Epochs`, and if so make it a singleton list `[Epochs]`. Then you can do the `average()`ing step the same way for all inputs.

Reply:
I don't think that works - the averaging only happens when they're lists of evoked. If it's a single epoch, it's epochs.iter_evoked.